lib/go/templates/templates.go

```diff
@@ -70,5 +72,11 @@ func replaceAddresses(code string, env Environment) string {
 		withHexPrefix(env.StakingProxyAddress),
 	)
 
+	code = strings.ReplaceAll(
```

Comment:
The goal is to make the templates so that they can be used by any language and not have language specific string formatting and make packages in each repo for each language. I'd love to look into a better format that is more universal, but this is what we'll use for now.